core/domain/exp_services.py

```diff
@@ -82,6 +82,9 @@ def _migrate_states_schema(versioned_exploration_states):
             exploration.
           - states: the dict of states comprising the exploration. The keys in
             this dict are state names.
+    Raise:
```

Comment:
FYI, I'm going to be a bit picky in this review, since this is meant to be a cleanup PR, which means that the result should have high standards of quality. So, I hope it's OK if I comment on a bunch of small things as well!

This should be "Raises". See https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments